lighthouse-core/formatters/critical-request-chains.js

```diff
@@ -159,9 +161,14 @@ class CriticalRequestChains extends Formatter {
 
         // Create the appropriate tree marker based on the depth of this
         // node as well as whether or not it has children and is itself the last child.
-        const treeMarker = ancestorTreeMarker +
-            (isLastChild ? '┗━' : '┣━') +
-            (hasChildren ? '┳' : '━');
+        let treeMarker = ancestorTreeMarker;
+        if (process.platform === 'win32') {
+          treeMarker += (isLastChild ? '\u2514\u2500' : '\u251C\u2500') +
```

Comment:
maybe do this as with `ancestorTreeMarker ` above, where the symbol is picked conditionally (maybe at file level) but all platforms construct the string in the same way?